docs/spec_draft.md

```diff
@@ -2234,6 +2235,39 @@ hidden state.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.round_nearest_afz
+
+### Semantics
+
+Performs element-wise rounding towards the nearest integer, breaking ties away
+from zero, on the `operand` tensor and produces a `result` tensor.
```

Comment:
Let's also add a second sentence: "Implements the \`roundToIntegralTiesToAway\` operation from the IEEE-754 specification".

Also, while we're at it, let's update the spec for `ceil` and `floor` to call out the names of the ops from IEEE-754, rather than their descriptions, i.e. "\`roundToIntegralTowardPositive\`" and "\`roundToIntegralTowardNegative\`"?